src/main/java/de/samply/share/broker/rest/Monitoring.java

```diff
@@ -248,13 +249,13 @@ public Response createQuery(@HeaderParam(HttpHeaders.AUTHORIZATION) String auth,
   public Response getResult(@HeaderParam(HttpHeaders.AUTHORIZATION) String auth,
       @PathParam("id") int queryId) {
     try {
-      if (auth == null || !checkBasicAuth(auth)) {
+      if (auth == null || !checkBasicAuth(auth,"icinga.username", "icinga.password")) {
```

Comment:
This is unrelated to the structured query feature. Please put that into it's own PR.

Reply:
So it is better to dublicate the code only to remove it later again?

Reply:
Doing so is the only way to be compatible with the current monitoring solution. Otherwise an update of the search broker would break the monitoring. The code is also not duplicated, because it's different.